modules/ROOT/pages/indexes/semantic-indexes/vector-indexes.adoc

```diff
@@ -371,13 +387,13 @@ The index name can also be given as a parameter, `DROP INDEX $name`.
 
 [role=beta]
 [[indexes-vector-set]]
-== Set a vector property on a node
+== Set a vector property
 
 Valid vectors for use in the index must have components finitely representable in {ieee-754} _single_ precision.
 They are represented as properties on nodes with the type `LIST<INTEGER | FLOAT>`.
-As of Neo4j 5.13, you can set a vector property using the {link-procedures-reference}#procedure_db_create_setNodeVectorProperty[`db.create.setNodeVectorProperty`] procedure.
+As of Neo4j 5.13, you can set a vector property on a node using the {link-procedures-reference}#procedure_db_create_setNodeVectorProperty[`db.create.setNodeVectorProperty`] procedure.
 It validates the input and sets the property as an array of {ieee-754} single precision values.
-This beta procedure replaces {link-procedures-reference}#procedure_db_create_setVectorProperty[`db.create.setVectorProperty`].
+This beta procedure replaces {link-procedures-reference}#procedure_db_create_setVectorProperty[`db.create.setVectorProperty`]. As of Neo4j 5.18, you can set a vector property on a relationship using the {link-procedures-reference}#procedure_db_create_setRelationshipVectorProperty[`db.create.setRelationshipVectorProperty`] procedure.
```

Comment:
```suggestion
This beta procedure replaces {link-procedures-reference}#procedure_db_create_setVectorProperty[`db.create.setVectorProperty`].
As of Neo4j 5.18, you can set a vector property on a relationship using the {link-procedures-reference}#procedure_db_create_setRelationshipVectorProperty[`db.create.setRelationshipVectorProperty`] procedure.
```
I assume docs will have a linebreak in between the sentences

Reply:
all of these `set` procedures are kept as "beta"